common/browser_options.go

```diff
@@ -60,10 +60,11 @@ func (l *LaunchOptions) Parse(ctx context.Context, opts goja.Value) error { //no
 		o  = opts.ToObject(rt)
 	)
 	for _, k := range o.Keys() {
-		var (
-			err error
-			v   = o.Get(k)
-		)
+		v := o.Get(k)
+		if v.Export() == nil {
+			continue // don't override the defaults on `null``
```

Comment:
Let's log a warning that we're falling back to a default value, and, if possible, what that value is.

Reply:
Welcome to the chain of PRs: #608 -> #616 -> Logger PR. These two are much better than the current `main`. I don't see any issues merging them. I'll send the next PR (logging) tomorrow.

Reply:
Please see #620.